positions.go

```diff
@@ -14,9 +14,45 @@ type Position struct {
 	SharesHeldForSells      float64 `json:"shares_held_for_sells,string"`
 }
 
+type OptionPostion struct {
+	Chain                    string        `json:"chain"`
+	AverageOpenPrice         string        `json:"average_open_price"`
+	Symbol                   string        `json:"symbol"`
+	Quantity                 string        `json:"quantity"`
+	Direction                string        `json:"direction"`
+	IntradayDirection        string        `json:"intraday_direction"`
+	TradeValueMultiplier     string        `json:"trade_value_multiplier"`
+	Account                  string        `json:"account"`
+	Strategy                 string        `json:"strategy"`
+	Legs                     []LegPosition `json:"legs"`
+	IntradayQuantity         string        `json:"intraday_quantity"`
+	UpdatedAt                string        `json:"updated_at"`
+	Id                       string        `json:"id"`
+	IntradayAverageOpenPrice string        `json:"intraday_average_open_price"`
+	CreatedAt                string        `json:"created_at"`
+}
+
+type LegPosition struct {
+	Id             string `json:"id"`
+	Position       string `json:"position"`
+	PositionType   string `json:"position_type"`
+	Option         string `json:"option"`
+	RatioQuantity  string `json:"ratio_quantity"`
+	ExpirationDate string `json:"expiration_date"`
+	StrikePrice    string `json:"strike_price"`
+	OptionType     string `json:"option_type"`
+}
+
+type Unknown interface{}
+
 // GetPositions returns all the positions associated with an account.
-func (c *Client) GetPositions(a Account) ([]Position, error) {
-	return c.GetPositionsParams(a, PositionParams{})
+func (c *Client) GetOptionPositions() ([]OptionPostion, error) {
+	return c.GetOptionPositionsParams(PositionParams{NonZero: true})
+}
+
+// GetPositions returns all the positions associated with an account.
+func (c *Client) GetPositions() ([]Position, error) {
```

Comment:
This looks like a breaking API change. Is there a specific reason to pull the Account out of the parameters?

Reply:
It looks like Robinhood used to return a string url on the account but now you can just call an endpoint for "positions" and Robinhood know who the caller is from the auth header. 

here is where that url was used 
https://github.com/andrewstuart/go-robinhood/blob/463e091050a8464f2b9ac41db3ad7182649dd655/positions.go#L41

but that returns blank and results in the method not returning anything.

TLDR; The Account param isnt needed and is not used